configure

```diff
@@ -1014,8 +1014,9 @@ my %all_sanitizers = (
     fsanitize => 'address',
     env => {
       LSAN_OPTIONS => "suppressions=$FindBin::RealBin/etc/asan-suppr.txt",
-      ASAN_OPTIONS => 'detect_leaks=1:fast_unwind_on_malloc=0',
+      ASAN_OPTIONS => 'detect_leaks=1:fast_unwind_on_malloc=0:strict_string_checks=1:detect_stack_use_after_return=1::check_initialization_order=1:strict_init_order=1',
```

Comment:
```suggestion
      ASAN_OPTIONS => 'detect_leaks=1:fast_unwind_on_malloc=0:strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1',
```
I assume `::` is not significant for asan

Reply:
I don't think so, but easy enough to fix.